### DISCUSSION TITLE
Approval for Event contributions

### DISCUSSION THREAD

**Discussion by zeel991** [egovernments/Digit-Core#579] [Say Hello]
Hello Everone!!
I am Zeel Darji , C4GT community Lead and Open source Lead of Development Wing of IIITN . I am organizing an open source event in my college's techfest which higlights the importance of DPGs and DPIs .

I wanted to ask you if participants could contribute to the repos of eGovernments for our Event ?
Thank You!

**Comment by Rishabh-egov** [comment]
Hey @zeel991 
Sure , go ahead. Please make sure that there are quailty contributions and no spams due to this.
